### PR TITLE
Port tests for pystiche.loss.functional 

### PR DESCRIPTION
**Pull request by sakshi-06**
This PR ports `pystiche.ops.functional` to   `pystiche.loss.functional`. Please see #514. 

This is my first PR so in case there are any mistakes please feel free to point them out, I will be looking forward to your review. 

cc: @pmeier

### CODE REVIEW

File: tests/integration/loss/test_functional.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import pytorch_testing_utils as ptu
+
+import torch
+from torch.nn.functional import mse_loss
+
+import pystiche.ops.functional as F
+
+
+class TestFunctionalLoss:
```
Comment (pmeier) [inline tests/integration/loss/test_functional.py:9]: Just seeing this: there is no need for this class here as the module name already implies that these are tests for the functionals. Plus, since it covers all test cases, this test case adds no value.

In order to keep the scope of the PR narrow, please revert this change and do a 1-to-1 port of the old file (minus my comment above). If there is something else here that needs to be addressed, we can do that in follow.up PRs.

File: tests/integration/loss/test_functional.py [hunk 2]
```diff
@@ -60,3 +60,4 @@ def get_checkerboard(size):
     actual = F.total_variation_loss(input)
     desired = 2.0
     assert desired == ptu.approx(actual)
+
```
Comment (pmeier) [inline tests/integration/loss/test_functional.py:63]:
```suggestion

```

**Comment by pmeier** [comment]
Thanks again @sakshi-06!
